includes/AMP/Output_Buffer.php

```diff
@@ -227,7 +225,7 @@ public function prepare_response( string $response ): string {
 		$dom = Document::fromHtml( $response );
 
 		if ( ! $dom instanceof Document ) {
-			return $this->render_error_page( SanitizationException::from_document_parse_error() );
+			return $this->render_error_page( \Google\Web_Stories\Exception\SanitizationException::from_document_parse_error() );
```

Comment:
😕

Reply:
Hmm this comes from when the `SlevomatCodingStandard.Namespaces.FullyQualifiedExceptions`  rule was initially enabled.

I have since removed it though, but the (automated) change was made before that though.

I don't mind it though.

Reply:
It is only one place, so let's leave it.